rascal2/ui/presenter.py

```diff
@@ -148,6 +149,15 @@ def ask_to_save_project(self):
 
         return proceed
 
+    def export_results(self):
+        """Export the results object."""
+        if self.model.results:
+            save_file = self.view.get_save_file("Export Results", "", "*.json")
+            if not save_file:
+                return
+
+            self.model.results.save(save_file)
```

Comment:
please wrap this in a `try except OSError` so we don't crash if the user saves in a place with wrong permissions. You could update the test also

```
  try:
      self.model.results.save(save_file)
  except OSError as err:
      self.view.logging.error(f"Failed to save project at path {save_file}:\n {err}")
```